src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Emitter/CoreBindingHelpers.cs

```diff
@@ -377,7 +383,8 @@ void EmitBindImplForMember(MemberSpec member)
                         member,
                         parsedMemberAssignmentLhsExpr,
                         sectionPathExpr: GetSectionPathFromConfigurationExpression(configKeyName),
-                        canSet: true);
+                        canSet: true,
+                        firstTimeInitialization: true);
```

Comment:
Would `InitializationKind.Declaration` express the same intent?

Reply:
Sure.